rholang/src/main/scala/coop/rchain/rholang/interpreter/Env.scala

```diff
@@ -1,33 +1,41 @@
 package coop.rchain.rholang.interpreter
 
-import scala.collection.immutable.HashMap
+import scala.collection.mutable
 
-/* Env reifies the Env[A] = HashMap[Int,A] type alias.
+/* Env reifies the Env[A] = mutable.LinkedHashMap[Int,A] type alias.
```

Comment:
Why mutable? I'm not strictly opposed, but please explain in a couple of sentences.

Reply:
Scala does not offer an immutable LinkedHashMap. The `LinkedHashMap` has O(1) access time while the alternative immutable `LinkedMap` has O(n) access time. Also, `TreeMap` and `SortedMap` both have O(log n) access time.